.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,23 @@
+name: Node.js CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [8.x, 9.x, 10.x, 11.x, 12.x, 13.x, 14.x]
```

Comment:
We're running our tests through karma so I'd imagine running it against multiple node versions won't matter too much (We're effectively testing via headless chrome regardless), but at some point if we write a suite directly in node then this could be useful.